src/tools/illink/src/ILLink.Tasks/ILLink.Tasks.csproj

```diff
@@ -20,6 +20,8 @@
          NU5129: Targets don't match the package id. This is intentional. -->
     <NoWarn>$(NoWarn);NU5128;NU5129</NoWarn>
     <IncludeBuildOutput>false</IncludeBuildOutput>
+    <!-- Place build output into a layout that matches the package layout. -->
+    <OutputPath>$(OutputPath)/tools</OutputPath>
```

Comment:
The build and package output paths are intentionally not aligned. Why do you need the tools subdirectory for the output path?

Reply:
We should keep the output path and package layout separate, otherwise we will hit other issues over time. Please revert the `OutputPath` change.